test/parallel/test-benchmark-timers.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const common = require('../common');
```

Comment:
Linter: `3:7  error  'common' is assigned a value but never used  no-unused-vars`